frontend/scripts/tests/table-modal/table-modal-selectors.test.js

```diff
@@ -45,30 +45,36 @@ describe('Table widget selectors', () => {
           chartType
         })
       ).toEqual([
-        { name: 'COMMODITY' },
-        { name: 'COUNTRY' },
-        { name: 'YEAR' },
-        { name: 'Trade volume', unit: 't' },
-        { name: 'Zero Deforestation Commitment (Exporter)', unit: '' }
+        { name: 'commodity' },
+        { name: 'country' },
+        { name: 'year' },
+        { name: 'trade volume', unit: 't', format: ',.2s' },
+        { name: 'zero deforestation commitment (exporter)' }
       ]);
     });
     // Multiple years, non-cont indicator, 1 source, no other flow path filters
-    xit('returns the headings for Stacked bar chart per year per value of non-continuous indicator - globally', () => {
+    it('returns the headings for Stacked bar chart per year per value of non-continuous indicator - globally', () => {
       const chartType = 'stackedBar';
       const { mYearNCont } = meta;
+      const oneSourceState = {
+        ...state,
+        sourcesPanel: {
+          activeItems: { 0: { name: 'Biome Name' } }
+        }
+      };
+
       expect(
-        getTableHeaders(state, {
+        getTableHeaders(oneSourceState, {
           meta: mYearNCont,
           data,
           chartType
         })
       ).toEqual([
-        { name: 'COMMODITY' },
-        { name: 'COUNTRY' },
-        { name: 'YEAR' },
-        { name: 'TRADE VOLUME', unit: 't' },
-        { name: 'BIOMES' },
```

Comment:
This is supposed to be only for the last case. That is only fixed with the last backend PR